### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by sinapsist**
Fix README.md and clean up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,6 +216,29 @@ Start and serve the dApp:
 npm run start
 ```
 
+### Hot Reload Contract Address 
+
+By default, every page refresh will deploy new contract and reset contract data
```
Comment (jchenche) [inline README.md:221]: This sentence is not really accurate. A page refresh will not perform the deployment per se. It updates the page UI to point to the address of the new deployment.

File: .taq/config.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "network": {
         "ghostnet": {
             "label": "ghostnet",
-            "rpcUrl": "https://rpc.ithacanet.teztnets.xyz",
+            "rpcUrl": "https://ghostnet.ecadinfra.com",
```
Comment (jchenche) [inline .taq/config.json:8]: Are you able to deploy with this new URL?

**Review by jevonearth — APPROVED** [review]
LGTM

File: .taq/config.json (hunk 2 above)
Comment (jevonearth) [reply .taq/config.json:8]: I copied the new config.json into my scaffolded project, and it appears to work fine.

File: README.md [hunk 3]
```diff
@@ -8,48 +8,70 @@
 | Frontend Dapp      | Yes                           |
 | Wallet Integration | Yes                           |
 
-## Quickstart
-
-In a rush? You can follow the steps below to get up and running immediatley:
-
-1. `taq scaffold https://github.com/ecadlabs/taqueria-scaffold-taco-shop taco-shop`
-2. `npm run setup`
```
Comment (jevonearth) [inline README.md:16]: I think we should restore this, and add a third step:
`npm run start:app`

Then add a new script entry to the root package.json that looks like this:
`"start:app": "cd app && npm run start"`

Then the react is running in a dev server.

**Review by jevonearth — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (sinapsist) [reply README.md:16]: Added.

File: README.md (hunk 1 above)
Comment (sinapsist) [reply README.md:221]: Thank you, Jimmy. Good point.

Comment (jevonearth) [reply README.md:221]: I'll fix this in a subsequent PR.

**Review by jevonearth — APPROVED** [review]

**Pull request merged**